luigi/notifications.py

```diff
@@ -254,19 +251,15 @@ def send_email_sendgrid(sender, subject, message, recipients, image_png):
     client.send(to_send)
 
 
-def _email_disabled():
+def _email_disabled_reason():
     if email().format == 'none':
-        logger.info("Not sending email when email format is none")
-        return True
+        return "Not sending email when email format is 'none'"
```

Comment:
Yeah, I tried my best not to change anything. I'm not sure why you would ever need none. Just don't set a receiver or an smtp server. Maybe this would be useful in a situation where you have multiple configs overriding parts of eachother?